src/ui/components/Tour/Tour.tsx

```diff
@@ -160,19 +160,31 @@ const Tour: React.FC = () => {
         )}
 
         {!isNewUser && showConsoleNavigate && showBreakpointAdd && showBreakpointEdit && (
-          <img src="/images/tour/fast-forward.gif" className={styles.videoExample} />
+          <img
+            src="https://website-git-master-recordreplay.vercel.app/tour/fast-forward.gif"
+            className={styles.videoExample}
+          />
         )}
 
         {!showConsoleNavigate && showBreakpointAdd && showBreakpointEdit && (
-          <img src="/images/tour/addlogs.gif" className={styles.videoExample} />
+          <img
+            src="https://website-git-master-recordreplay.vercel.app/tour/addlogs.gif"
+            className={styles.videoExample}
+          />
         )}
 
         {!showConsoleNavigate && !showBreakpointAdd && showBreakpointEdit && (
-          <img src="/images/tour/editlogs.gif" className={styles.videoExample} />
+          <img
+            src="https://website-git-master-recordreplay.vercel.app/tour/editlogs.gif"
+            className={styles.videoExample}
+          />
         )}
 
         {hasCompletedTour && (
-          <img src="/images/tour/consoleupdate.gif" className={styles.videoExample} />
+          <img
+            src="https://website-git-master-recordreplay.vercel.app/tour/consoleupdate.gif"
```

Comment:
Two topics here:

1. I'm pointing to vercel but I need to change this to static.replay
2. But Ryan did suggest pointing to static.replay because I'm about to put a ton of videos and images into our app, and it's better to host them in a static place rather than bloating our package

Reply:
I'm not sure how static.replay works. (I guess that's where we deploy our protocol docs?) But I guess that's fine if that's what Ryan suggested. We shouldn't be pointing to website-git-master-recordreplay.vercel.app though.